src/protobuf-net.Reflection/CSharpCodeGenerator.cs

```diff
@@ -141,7 +141,7 @@ protected override void WriteFileHeader(GeneratorContext ctx, FileDescriptorProt
                .WriteLine("// </auto-generated>")
                .WriteLine()
                .WriteLine("#region Designer generated code")
-               .Write($"#pragma warning disable {prefix}0612, {prefix}0618, {prefix}1591, {prefix}3021");
+               .Write($"#pragma warning disable {prefix}0612, {prefix}0618, {prefix}1591, {prefix}3021, {prefix}8981");
```

Comment:
also L171, sorry

Reply:
they *are*, and it isn't strictly required - unlike a few other things like `#if` - maybe I'm just OCD :p